### PR TITLE
Make sure default credentials are used if none supplied

### PR DESCRIPTION
**Pull request by zentron**

### CODE REVIEW

File: Server/Scripts/configure.ps1 [hunk 1]
```diff
@@ -25,15 +26,15 @@ $env:OCTOPUS_INSTANCENAME=$OctopusInstanceName
...
     {
       if ($octopusAdminUsername -eq $null)
       {
-        $octopusAdminUsername = "admin"
+        $script:octopusAdminUsername = "admin"
       }
       if ($octopusAdminPassword -eq $null)
       {
-        $octopusAdminPassword = "Passw0rd123"
+        $script:octopusAdminPassword = "Passw0rd123"
```
Comment (zentron) [inline Server/Scripts/configure.ps1]: the `$script:` prefix is the key change. Previously the default wouldnt be applied

Comment (stopthatastronaut) [reply Server/Scripts/configure.ps1]: Good old PowerShell scoping. There's often a more elegant way than explicit scoping like this - like maybe returning a hashtable out of this function to keep the scopes tight - but honestly, if it works, I'm fine with it

File: Scripts/common.ps1 [hunk 2]
```diff
@@ -28,9 +28,17 @@ function Write-Log
   Write-Host "[$timestamp] $message"
 }
 
-function Execute-Command ($exe, $arguments)
+function Execute-Command ($exe, $arguments, $mask)
 {
-  Write-Log "Executing command '$exe $($arguments -join ' ')'"
+  $maskedArgs = $($arguments -join ' ')
+  if ($mask -ne $null) {
```
Comment (stopthatastronaut) [inline Scripts/common.ps1]: PSScriptAnalyzer would whinge here about putting null on the right when it's better on the left. But as with other comments, it works for now, and we're not currently running PSScriptAnalyzer on it.

File: Server/Scripts/configure.ps1 [hunk 3]
```diff
@@ -11,6 +11,7 @@ $env:OCTOPUS_INSTANCENAME=$OctopusInstanceName
   $masterKeySupplied = ($masterKey -ne $null) -and ($masterKey -ne "")
   $octopusAdminUsername = $env:OctopusAdminUsername
   $octopusAdminPassword = $env:OctopusAdminPassword
+  $credentialsSupplied=($octopusAdminPassword -ne $null -or $octopusAdminUserName -ne $null)
```
Comment (stopthatastronaut) [inline Server/Scripts/configure.ps1]: another nulls on the left thing. Can be addressed later if we put Pester testing on this repo

**Pull request merged**